lib/README.md

```diff
@@ -240,3 +237,6 @@ ExternalLinkContentViewerUI.jar
   now being carried in this jar file instead of source until we migrate to Java 1.6
   No longer used as of JMRI 2.99.1
   
+servlet.jar:
```

Comment:
Not sure what you're suggesting.  The servlet.jar file was removed, so there's now a line in the "Older, no longer present:" section of the doc to say that.  (CATS and a couple user programs have copied the JMRI launchers, so we keep a list of the old-no-longer-present files to help them maintain them)